app/services/key_rotator.rb

```diff
@@ -0,0 +1,8 @@
+module KeyRotator
+  extend ActiveSupport::Concern
+
+  def old_keys(config_name)
+    queue_name = "#{config_name}_queue"
+    JSON.parse(Figaro.env.send(queue_name.to_sym) || '[]')
```

Comment:
smallest nit/idea: would it be less indirection if we inlined the symbol interpolation:

```
JSON.parse(Figaro.env.send(:"{config_name}_queue") || '[]')
```